app/templates/events/event_list.html

```diff
@@ -88,8 +89,11 @@ <h2 class="accordion-header" id="accordion__header_{{program}}">
                       <td>{{event.timeStart.strftime('%I:%M %p')}}</td>
                       <td>{{event.location}}</td>
                       {% if (user.isCeltsAdmin) or (user.isCeltsStudentStaff) %}
-                        <td><i class="bi bi-envelope-fill"> Email</i></td>
-                        {# FIXME: the Email icon should be clickable and lead to email template #}
+                        <td>
+                            <button class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#emailModal" onclick="passEventInfo({{event.id}}, {{program.id}}, {{selectedTerm}})">
+                              <span class="bi bi-envelope-fill"> Email</span>
+                            </button>
+                        </td>
                       {% else %}
                         {% if (event.startDate < currentTime.date()) or (event.timeStart < currentTime.time() and event.startDate == currentTime.date())%}
```

Comment:
Let's make this logic a property of the Event model, so we can use it like so: `if event.isPast:`